packages/azure-kusto-data/package.json

```diff
@@ -50,7 +51,8 @@
         "dist-esm/src/*.js.map",
         "types/src/*.d.ts",
         "tsconfig.tsbuildinfo",
-        "dist-esm/src/wellKnownKustoEndpoints.json"
+        "dist-esm/src/wellKnownKustoEndpoints.json",
+        "package.json"
```

Comment:
why

Reply:
Oops, this was for getting the version from the package.json, shouldn't be from this branch